dotcom-rendering/src/components/StickyBottomBanner.importable.tsx

```diff
@@ -292,7 +289,7 @@ export const StickyBottomBanner = ({
 			idApiUrl,
 		});
 		const brazeArticleContext: BrazeArticleContext = {
-			section: sectionName,
+			section: sectionId,
```

Comment:
`BrazeArticleContext` is defined in an npm library, so I'm not changing that field name now